troi/utils.py

```diff
@@ -2,12 +2,30 @@
 import inspect
 import os
 import traceback
+import sys
 
 from troi import Element, Artist, Release, Recording
 import troi.patch
 
 
-def discover_patches(patch_dir):
+def discover_patches():
+    """
+        Attempt to load patches from the installed patches dir as well as any patches directory in the current dir.
+    """
+
+    patches = discover_patches_from_dir("troi.patches.", os.path.join(os.path.dirname(__file__), "patches"))
```

Comment:
it's a bit annoying that you have to specify this twice. Any thoughts on how you could specify it just once?
For example you could do:

```
>>> patches = importlib.import_module("troi.patches")
>>> patches.__file__
'/Users/alastair/code/metabrainz/troi-recommendation-playground/troi/patches/__init__.py'
```

which you could then use os.path.dirname in order to get the dir to use `listdir`.
This requires that it's a module (i.e. has `__init__.py`)

Reply:
Mildly annoying yes. Not worth sticking more time into it -- I've already spent loads of time on this code and don't really feel like improving tiny bits like this.